exercise/templates/exercise/_children.html

```diff
@@ -9,13 +9,13 @@
 {% elif entry.up %}
 </ul>
 {% endif %}
-{% elif entry|is_listed %}
+{% elif entry|is_listed and not entry|is_in_maintenance %}
```

Comment:
Did you consider changing the definition of the `is_listed` template tag? It would be easy to check the status there.

https://github.com/apluslms/a-plus/blob/ae4315c0b5285936353a501325764ef1b2d726b3/exercise/cache/hierarchy.py#L305

Reply:
Good idea!